app/src/main/java/com/labnex/app/activities/ProfileActivity.java

```diff
@@ -99,25 +99,30 @@ public void onResponse(
 									}
 								}
 
-								if (!user.getDiscord().isEmpty()
-										|| !user.getLinkedin().isEmpty()
-										|| !user.getTwitter().isEmpty()) {
+								boolean hasDiscord =
+										user.getDiscord() != null && !user.getDiscord().isEmpty();
+								boolean hasLinkedin =
+										user.getLinkedin() != null && !user.getLinkedin().isEmpty();
+								boolean hasTwitter =
+										user.getTwitter() != null && !user.getTwitter().isEmpty();
+
+								if (hasDiscord || hasLinkedin || hasTwitter) {
 									binding.socialInfo.setVisibility(View.VISIBLE);
 
-									if (!user.getDiscord().isEmpty()) {
+									if (hasDiscord) {
```

Comment:
You can check inline with `user.getDiscord() != null && !user.getDiscord().isEmpty()`, this will reduce adding three extra lines for the boolean declaration.

Reply:
I did it this way to avoid having a huge outer `if` block checking all three and redoing the lookups inside.  I've updated it now to inline them, though I'm not sure I particularly like spotless' formatting of it.  🙃